### PR TITLE
Added presentation for why you should contribute to FOSS

### PR DESCRIPTION
**Pull request by OmairK**
I was also thinking about changing the theme and some slides are repetitive like the Mozilla mission and your introduction slide. What do you suggest? @CuriousLearner

### CODE REVIEW

**Review by CuriousLearner — COMMENTED** [review]
Looks good overall, there are some minor changes though. 

In some of the slides there are full-stops multiple times.

Thanks for all your work :)

File: why_foss/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# FOSS
```
Comment (CuriousLearner) [inline why_foss/README.md]: This ReadMe should be changed to `Why should I contribute to Open Source` and relevant blog post link can be added to the description.

**Comment by CuriousLearner** [comment]
The repetitive slides are okay. The idea is that these things should be standalone and not dependent on each other :)

**Comment by OmairK** [comment]
Okay I am going to update the ReadMe. You want to keep the same theme,
right?

**Comment by CuriousLearner** [comment]
You can change the theme if you want to ;)

**Comment by OmairK** [comment]
Well it looks nice to me :p

File: why_foss/README.md (hunk 1 above)
Comment (OmairK) [reply why_foss/README.md]: I updated the ReadMe file and also added the link to your blog.

**Pull request closed without merging**